### DISCUSSION TITLE
Amplitude scale

### DISCUSSION THREAD

**Discussion by alexgrusu** [llfbandit/record#254] [General]
I tried to identify the scale of the amplitude but I didn't find anything related yet.

Any thoughts about the scale? Is there a difference between `iOS` and `Android`?

On silence, the current value for `Amplitude` is `-160.0`.

**Comment by llfbandit** [comment]
Amplitude is expressed as dB FS.
Max is 0 and min is locked at -160.

**Reply by alexgrusu** [reply]
And this applies to both iOS and Android?
